sqlmesh/core/engine_adapter/base.py

```diff
@@ -453,16 +483,17 @@ def create_view(
 
         if properties.expressions:
             create_kwargs["properties"] = properties
-
-        self.execute(
-            exp.Create(
-                this=schema,
-                kind="VIEW",
-                replace=replace,
-                expression=query_or_df,
-                **create_kwargs,
-            )
-        )
+        for source_query in source_queries:
```

Comment:
If we know that there should only be one query, shouldn't we just extract it and avoid an extra layer of nestedness?

Reply:
Yes good point. Will fix.